lib/presentation/app.dart

```diff
@@ -32,9 +32,10 @@ class VostApp extends StatelessWidget {
               title: 'Vost App',
               bloc: DependencyProvider.of(context).getHomeBloc(),
             ),
-        routeSplash: (context) => SplashPage(title: 'Splash Page'),
+        routeSplash: (context) => SplashPage(title: 'Splash Page', bloc: DependencyProvider.of(context).getSplashBloc(),),
```

Comment:
Shouldn't this be on internalization support?

Reply:
Same as the other issue - this will be done when all the copy is given